src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
-spring.datasource.password=mysecretpassword
-spring.datasource.url=jdbc:postgresql://localhost:5432/spring
+spring.datasource.password=599286666d
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
Avoid commit secret password to the repo...
And if you commit changes in the properties, please let us know what changes you made, because each one of us has different  setup